homeassistant/components/upnp/config_flow.py

```diff
@@ -216,9 +216,7 @@ async def async_step_ssdp(self, discovery_info: ssdp.SsdpServiceInfo) -> FlowRes
                 # SETUP_RETRY, another load is started,
```

Comment:
Shouldn't the config entries helper handle this case? We should just use`self.async_update_reload_and_abort` I think.